### ISSUE TITLE
[BUG] Layout issues with alt text on videos

### ISSUE THREAD

**Issue by impliedgg** [mastodon/mastodon-ios#1086]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The top video controls (close, view, volume) are next to inaccessible when you open a video in full screen view (tap the video). The alt text button overlays them.

### Expected Behavior

The full screen controls are accessible and the alt text is still viewable.

### Steps To Reproduce

1. Find a video with alt text.
2. Tap on the video to pop open the video viewer.
3. Try to press the controls on the top of the screen.

### Environment

```markdown
- Device: iPhone 6s+ (8,2)
- OS: 15.7.2
- Version: v2023.10
- Build: 333
```

### Anything else?

Apologies for using a meme, but this is where I first encountered this issue. Post from screenshot is https://tech.lgbt/@micah/110651790382481187
![F1BC0BCF-69ED-4CDE-918B-1D4006BB3846](https://github.com/mastodon/mastodon-ios/assets/58041173/ed7431d7-5813-4961-8a52-65aaa3f72d62).

**Comment by zeitschlag** [comment]
Thank you for this report. This should be fixed in `2023.11`, so please update the App from the App Store and feel free to close this issue afterwards :)

**Comment by impliedgg** [comment]
Appears to be fixed, closing now.
